src/playground/with-layout/chat/chat-custom-message-showcase.component.html

```diff
@@ -0,0 +1,48 @@
+<nb-chat title="Nebular Conversational UI" size="large">
+    <nb-chat-message *ngFor="let msg of messages"
+        [type]="msg.type"
+        [message]="msg.text"
+        [reply]="msg.reply"
+        [sender]="msg.user.name"
+        [date]="msg.date"
+        [files]="msg.files"
+        [quote]="msg.quote"
+        [latitude]="msg.latitude"
+        [longitude]="msg.longitude"
+        [avatar]="msg.user.avatar"
+        [customMessageData]="msg.customMessageData">
+
+        <div *nbCustomMessage="'link'; let customMessageData">
+            <a [href]="customMessageData.href">{{ customMessageData.label }}</a>
+        </div>
+        <div style="padding-top: 5px;" *nbCustomMessage="'button'; let customMessageData">
+            <button style="width: 100%; height: 100%;">{{ customMessageData }}</button></div>
+        <div *nbCustomMessage="'table'">
+            <table>
+                <tr>
+                    <th>Firstname</th>
+                    <th>Lastname</th>
+                    <th>Age</th>
+                </tr>
+                <tr>
+                    <td>Robert</td>
+                    <td>Baratheon</td>
+                    <td>46</td>
+                </tr>
+                <tr>
+                    <td>Jaime</td>
+                    <td>Lannister</td>
+                    <td>31</td>
+                </tr>
+            </table>
+        </div>
+        <div *nbCustomMessage="'img'">
+            <picture>
+                <img src="https://googlechrome.github.io/samples/picture-element/images/kitten-small.png" alt="a cute kitten">
+            </picture>
+        </div>
```

Comment:
Very similar to the link example above, I think this one could be removed to not overload the example.